i18n-helpers/src/bin/mdbook-gettext.rs

```diff
@@ -90,7 +90,7 @@ fn translate_book(ctx: &PreprocessorContext, mut book: Book) -> anyhow::Result<B
 
     let catalog = po_file::parse(&path)
         .map_err(|err| anyhow!("{err}"))
-        .with_context(|| format!("Could not parse {:?} as PO file", path))?;
+        .with_context(|| format!("Could not parse {path:?} as PO file"))?;
```

Comment:
Can I get you to submit this to https://github.com/google/mdbook-i18n-helpers/ instead? I moved the code to a stand-alone repository yesterday.

Reply:
I'm removing the code you see here in #552 which should be merged later today, hopefully :smile:

Reply:
@nyurik the PR was merged and this caused a few conflicts here... sorry about that. I hope you can revert the changes in your PR (and probably take out the controversial mixed cases as well just to make things simple).

Reply:
Thanks for updating the PR!